moveit_configs_utils/default_configs/ompl_planning.yaml

```diff
@@ -0,0 +1,9 @@
+planning_plugin: ompl_interface/OMPLPlanner
```

Comment:
Any reason to split this from the other ompl parameters file?

Reply:
So if there's one file, the options are
 * Use the unified `default_configs/ompl_planning.yaml` options
 * Specify your own `ompl_planning.yaml`. 

Each of these parameters will have the standard parameters (`planning_plugin` and `request_adapters`) and all the `planner_configs`. 

By splitting the `planner_configs` into their own yaml that is loaded via special rule, the options are more flexible. 
 * Use all the ompl parameters from `default_configs`
 * Specify our own `ompl_planning.yaml` with changes to `request_adapters` etc. 
 * Specify our own `ompl_planning.yaml` that also includes `planner_configs`. 

Hence, if we just wanted to add a `request_adapter` or change some other base parameter (outside the `planner_configs`), we don't have to manually copy all the `planner_configs` to our new parameter file.